src/boot.lisp

```diff
@@ -585,4 +585,18 @@
 (defmacro print-unreadable-object ((object stream &key type identity) &body body) 
     `(!print-unreadable-object (,object ,stream :type ,type :identity ,identity) ,@body))
 
+(defmacro %%assert (test &optional ignore datum &rest args)
+  (let ((value (gensym "ASSERT-VALUE"))
+        (name (gensym "ASSERT-BLOCK")))
+    `(block
```

Comment:
That's interesting.  Just curious, why is the block needed if we don't use `name` anywhere?

Reply:
>That's interesting. Just curious

;-)

And indeed
There is more than one such place throughout the all code
But not now, not now
I'm note that